frontend/dashboard/components/RepoList/RepoList.module.css

```diff
@@ -1,53 +1,48 @@
-.actionLink {
-  align-items: center;
-  display: flex;
-  margin-right: 1rem;
-  min-width: 40px;
+.favoriteIconHeaderCell {
+  font-size: 0; /*Hides text, but keeps it accessible for screen readers*/
+  width: 46px; /*Has to be exactly 46px to make the width the same for empty and filled table columns*/
 }
 
-.actionLink:hover {
-  text-decoration: none;
+.nameHeaderCell {
+  width: 20%;
 }
 
-.actionLink:hover span {
-  text-decoration: underline;
+.createdByHeaderCell {
+  width: 20%;
 }
 
-.dropdownIcon {
-  font-size: 2rem;
+.lastUpdatedHeaderCell {
+  width: 15%;
```

Comment:
I feel we could utilize the grid-property here, and assign fractions instead 🤔

Reply:
Good idea!
Is it OK to target the HTML elements directly like this? The table keeps breaking when trying to implement specific classNames like `.tableRows` and `.tableCells`

````
tr {
  display: grid;
  grid-template-columns: 4.5rem 1fr 1fr 0.5fr 1fr 11rem;
}

td {
  display: flex;
  align-items: center;
}
````